### PR TITLE
kanidm: 1.1.0-beta.13 -> 1.1.0-rc.15

### PR DESCRIPTION
**Pull request by adamcstephens**
## Description of changes

https://github.com/kanidm/kanidm/releases/tag/v1.1.0-rc.15

~~Currently failing a test, so not ready yet.~~ Maintainers feel free to edit my PR if I'm not around.

Failing test disabled upstream.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by adamcstephens** [comment]
This is now passing and ready for review.

**Comment by Kranzes** [comment]
@ofborg test kanidm

File: pkgs/servers/kanidm/default.nix [hunk 1]
```diff
@@ -19,17 +19,23 @@ let
...
-  version = "1.1.0-beta.13";
+  version = "1.1.0-rc.15";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
-    # Latest 1.1.0-beta.13 tip
-    rev = "5d1e2f90e6901017ab3ef9b5fbc10e25a5451fd2";
-    hash = "sha256-70yeHVOrCuC+H96UC84kly3CCQ+y1RGzF5K/2FIag/o=";
+     # Latest 1.1.0-rc.15 tip
```
Comment (Flakebi) [inline pkgs/servers/kanidm/default.nix:27]: Spurious space before this comment?

**Review by Flakebi — APPROVED** [review]
Looks good, thanks!

Test runs through (`nix-build -A kanidm.tests`) and seems to work fine on my system.

**Review by Flakebi — APPROVED** [review]
Thanks, still LGTM

**Review by erictapen — APPROVED** [review]
LGTM

**Comment by Kranzes** [comment]
Are CI checks screwed up? Should I merge anyway?

**Comment by adamcstephens** [comment]
> Are CI checks screwed up? Should I merge anyway?

I hate to merge failing builds. I'm re-running these locally to test and see if I can reproduce the failure.

**Comment by adamcstephens** [comment]
Both x86_64 and aarch64 tests run successfully me.

The errors from the ofborg failures seem to be:

```
server # [    5.189970] (kanidmd)[846]: kanidm.service: Failed to set up mount namespacing: /run/systemd/mount-rootfs/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/ofborg-evaluator-1/nixos/tests/common/acme/server: No such file or directory
server # [    5.192497] (kanidmd)[846]: kanidm.service: Failed at step NAMESPACE spawning /nix/store/wqpcn0626p7a6sgpn2bc3aispsqa90v1-kanidm-1.1.0-rc.15/bin/kanidmd: No such file or directory
```

**Comment by mweinelt** [comment]
Some hardening option on the systemd unit interferes with test runs on ofborg infra. This is not news, has happened on every update to kanidm before this one as well.

**Comment by Kranzes** [comment]
Shall we merge?

**Comment by Flakebi** [comment]
> Shall we merge?

Yes, please. From what @mweinelt says, it’s definitely not a regression (and the test passes fine outside of ofborg).

**Pull request merged**